src/builder/QuarkBuilder.sol

```diff
@@ -560,6 +686,15 @@ contract QuarkBuilder {
         Actions.Action[] memory actions,
         Accounts.ChainAccounts[] memory chainAccountsList,
         uint256 targetChainId
+    ) internal pure {
+        return assertSufficientPaymentTokenBalances(actions, chainAccountsList, targetChainId, 0);
+    }
+
+    function assertSufficientPaymentTokenBalances(
```

Comment:
Would be helpful to document what the `supplementalPaymentTokenBalance` is and how it is used.

Reply:
added a `@dev` comment